docs/support/Troubleshooting.md

```diff
@@ -1,46 +1,44 @@
+---
+sidebar_position: 2
+---
+
 # Troubleshooting
```

Comment:
I think this section should be called "Common Problems". I'd be more included to click that than the title "Troubleshooting". That could be just me though.

Reply:
Yeah, if it would not offend, I'm inclined to keep it as "Troubleshooting". I played around with changing it but as I started to look at where other docs were linking to it, it didn't feel right.